src/workloads/issues/ConnectionPoolStress.yml

```diff
@@ -41,9 +41,11 @@ Constants:
 
 Clients:
   Default:
+    PreWarm: false
     QueryOptions:
       maxPoolSize: *GlobalMaxPoolSize
   PerThread:
+    PreWarm: false
```

Comment:
I have a fairly strong preference for `NoPreWarm: true`. I feel that with bools you should try and keep the default option `false`